composer.json

```diff
@@ -166,15 +166,13 @@
             "Claroline\\BundleRecorder\\ScriptHandler::buildBundleFile"
         ],
         "pre-install-cmd": [
-            "php bin/configure",
-            "php bin/check"
+            "php bin/configure"
         ],
         "post-install-cmd": [
             "composer bundles"
         ],
         "pre-update-cmd": [
             "php bin/configure",
-            "php bin/check",
```

Comment:
Does this mean that we should remove this from the Docker files as well?

Reply:
Yes, I will do it. This is useless because it just checks for node/npm versions, which are already fixed by `package.json`.

Reply:
I've also removed some useless calls to `bin/configure` in the docker configs as they are done by a composer hook on pre-install and pre-update.